### PR TITLE
reviewVoteScore migration

### PR DESCRIPTION
**Pull request by Raemon**
This is an initial version of a voteMigration script. It doesn't do all the things we might want it to do (i.e. doesn't do any fancy normalization).

I think it's good enough for the Preliminary Vote phase but we need to figure something better out for the Final Voting Phase.

### CODE REVIEW

**Comment by Raemon** [comment]
Just realized this doesn't distinguish between 1000 karma and regular voters

**Comment by Raemon** [comment]
When I console log the userIds... some of them have quotes around them and some don't and I'm weirded out:

image

**Comment by Raemon** [comment]
image

**Comment by jpaddison3** [comment]
@raemon777 I bet the ones with quotes around them start with numbers instead of letters.

File: packages/lesswrong/server/migrations/2021-12-13-updateQuadraticVotes.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        }
+        if (user.groups?.includes('alignmentForum')) {
+          updatePost(postsAFUsers, vote)
+        }
+      })
+      // eslint-disable-next-line no-console
+      console.log(userId, totalUserPoints, totalUserPoints > 500 ? "Over 500" : "")
+    })
+
+    Object.keys(postsAllUsers).forEach(async postId => {
```
Comment (jpaddison3) [inline packages/lesswrong/server/migrations/2021-12-13-updateQuadraticVotes.ts:87]: Never use async in a `forEach`:

https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop

**Pull request merged**